pkg/controller/clusterprovision/clusterprovision_controller.go

```diff
@@ -241,12 +245,12 @@ func (r *ReconcileClusterProvision) createJob(instance *hivev1.ClusterProvision,
 		return reconcile.Result{}, err
 	}
 
-	return reconcile.Result{}, nil
+	return reconcile.Result{RequeueAfter: delayAfterInstallJobCreation}, nil
```

Comment:
This is not going to do quite what I suspect you want it to. This will requeue the ClusterProvision in 60 seconds. However, changes to the Job will cause the ClusterProvision to be requeued earlier.